packages/cli/src/__tests__/artificial-panic.test.ts

```diff
@@ -131,14 +131,14 @@ describe('artificial-panic get-config', () => {
   })
 })
 
-describeIf(process.env.PRISMA_CLI_QUERY_ENGINE_TYPE == 'library')('artificial-panic library', () => {
+describe('artificial-panic get-dmmf', () => {
```

Comment:
```suggestion
describe('artificial-panic get-dmmf (wasm)', () => {
```
maybe? just to make it clear

Reply:
Or maybe just a comment at the top of the file